packages/core-cairo/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@openzeppelin/wizard-cairo",
-  "version": "0.17.0",
+  "version": "0.18.0",
```

We could revert this and bump when a release is needed
```suggestion
  "version": "0.17.0",
```